src/network/dandelion.py

```diff
@@ -47,15 +49,31 @@ def poissonTimeout(start=None, average=0):
             average = FLUFF_TRIGGER_MEAN_DELAY
         return start + expovariate(1.0 / average) + FLUFF_TRIGGER_FIXED_DELAY
 
+    def init_pool(self, pool):
+        """pass pool instance"""
+        self.pool = pool
+
+    def is_dandelion_enabled(self, config):
+        """Check if Dandelion is enabled and set value in enabled attribute"""
+        dandelion_enabled = config.safeGetInt('network', 'dandelion')
+        # dandelion requires outbound connections, without them,
+        # stem objects will get stuck forever
+        if dandelion_enabled and not config.safeGetBoolean(
```

Comment:
We probably don't need to check for `dandelion_enabled` here.

Reply:
Value is coming from config  at startup. so i think it is required.

Reply:
I mean we're checking twice, should work ok if we just check once.